src/code/z_actor_dlftbls.c

```diff
@@ -37,6 +39,23 @@
 #define DEFINE_ACTOR_INTERNAL(name, _1, allocType, nameString) \
     { 0, 0, NULL, NULL, NULL, &name##_InitVars, nameString, allocType, 0 },
 
+#else
+
+#define DEFINE_ACTOR(name, _1, allocType, _3)  \
```

Comment:
could you add a comment like
`// non-debug sets ActorOverlay.name to NULL`
or similar